src/states/LevelOne.js

```diff
@@ -5,12 +5,16 @@ var LevelOne = function () {
   this.returnBtn = null;
   this.sceneBtn = null;
   this.scene = null;
+  this.backgroundImg = null;
 };
 
 LevelOne.prototype = {
 
   preload: function() {
     game.stage.backgroundColor = '#444444';
+    this.backgroundImg = game.add.image(0, 0, game.cache.getBitmapData('background'));
+    this.backgroundImg.width *= 2;
```

Comment:
This should probably read `this.backgroundImg.width = game.width` that way if you make a change to the size in worldGenerator you won't have to remember to change this.